wallet/wallet.go

```diff
@@ -2522,6 +2522,10 @@ type GetTransactionResult struct {
 func (w *Wallet) GetTransaction(txHash chainhash.Hash) (*GetTransactionResult,
 	error) {
 
+	var bestHeight int32
+	var blockHash *chainhash.Hash
+
+	var summary *TransactionSummary
```

Comment:
nit: can unify all these into a single `var ( ... )` block.

Reply:
Updated in updated commit